rfi_file_monitor/operations/dummy_operation.py

```diff
@@ -9,12 +9,13 @@
 
 from ..operation import Operation
 from ..utils.exceptions import SkippedOperation
-from ..utils.decorators import with_pango_docs
-from ..file import File
+from ..utils.decorators import with_pango_docs, supported_filetypes
+from ..file import File, RegularFile, AWSS3Object
 
 logger = logging.getLogger(__name__)
 
 @with_pango_docs(filename='dummy_operation.pango')
+@supported_filetypes(filetypes=[RegularFile, AWSS3Object])
```

Comment:
I guess this decorator will also need to go on the other classes, like sftp uploader... Also in the extensions

Reply:
If they only support `RegularFile`, then the decorator is not necessary.